### ISSUE TITLE
Not an Issue but a thanks.

### ISSUE THREAD

**Issue by METHF6THER** [IMarkoMC/Tokens#20]
The bot that the token connected to this was here for wasn't used for primarily anything important, but I could only imagine what the actual bot developers were doing. Thanks for making this a real thing! I'm sure plenty of people will love this. :)

**Comment by VictorTheDuck** [comment]
Same here. Imagine the amount of pure human suffering these people are saving from scammers.

**Comment by skidmaster69420** [comment]
Hello there,

The flies here rejoiced that their work is being appreciated, we rest assured that the repository owner that we work for is very happy as well.

There are no more important FND news, I have been released from my duties to include them in my posts. Also, with the ongoing peace. The flies not only gave me Solitare back but also hired a Solitare advanced professor who has taught Solitare in Solitary confinement in which a 0IQ Mosquito managed to learn it from them. Hence the professor had SEVERE ISSUES teaching me saying the mosquito knew more than me. I was very upset but with 69 hours of constant nonstop training, I eventually managed to win 1 game in every 100. 

Yours,
Paid Intern
SKIDMASTER 69420LLC
